code/modules/materials/_materials.dm

```diff
@@ -91,9 +91,9 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 
 	// Icons
 	var/icon_base = 'icons/turf/walls/solid.dmi'
-	var/icon_stripe = 'icons/turf/walls/stripes.dmi'
 	var/icon_base_natural = 'icons/turf/walls/natural.dmi'
 	var/icon_reinf = 'icons/turf/walls/reinforced_metal.dmi'
+	var/wall_paintable = 0 //PAINT_PAINTABLE|PAINT_STRIPABLE
```

Comment:
What's the purpose of the comment here?

Reply:
Was to outline the possible flags. Removed.